make/test.mk

```diff
@@ -80,6 +80,7 @@ ifeq ($(OPENSHIFT_VERSION),3)
 endif
 	$(eval package_yaml := ./manifests/devconsole/devconsole.package.yaml)
 	$(eval devconsole_version := $(shell cat $(package_yaml) | grep "currentCSV"| cut -d "." -f2- | cut -d "v" -f2 | tr -d '[:space:]'))
+	$(Q)cp ./deploy/crds/*.yaml ./manifests/devconsole/$(devconsole_version)/
```

Comment:
do we still need the test/sanitychecks/crd_test.go after this?

Reply:
Good catch!
The test is failing now.
I will remove it.

Reply:
Done. 07f351b3cd03fade0525ad7320d86a6cca56691a